src/scrabble.js

```diff
@@ -1,5 +1,55 @@
-function scrabble() {
-
+function scrabble(word) {
+  const lettersAN = {
```

Comment:
Object use! Yeaaaah! I also like how you had it in alphabetical order... (rather than points order) much easier as a human to go through and find the relevant score for each letter.